engine/src/main/java/org/camunda/bpm/engine/impl/cmd/TelemetryConfigureCmd.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.impl.cmd;
+
+import org.camunda.bpm.engine.impl.interceptor.Command;
+import org.camunda.bpm.engine.impl.interceptor.CommandContext;
+import org.camunda.bpm.engine.impl.persistence.entity.AuthorizationManager;
+
+public class TelemetryConfigureCmd implements Command<Object> {
+
+  protected boolean telemetryEnabled;
+
+  public TelemetryConfigureCmd(boolean telemetryEnabled) {
+    this.telemetryEnabled = telemetryEnabled;
+  }
+
+  public Object execute(CommandContext commandContext) {
+
+    AuthorizationManager authorizationManager = commandContext.getAuthorizationManager();
+    authorizationManager.checkCamundaAdmin();
+
+    new TelemetrySetupCommand(telemetryEnabled).execute(commandContext);
```

Comment:
Since we discussed to always insert the property into the database on engine startup, it would be more performant to just update the property and not go again through the exclusive lock business logic. Also, no `OptimisticLockingException` is thrown with the exclusive lock approach which is counterintuitive for the user.